packs/feats/class/kineticist/rising-hurricane.json

```diff
@@ -12,7 +12,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>A hurricane lifts your enemies into the air before bringing them crashing down in a bloody rain. Your hurricane appears in a cylinder that's 40 feet tall, has a 30-foot diameter, and is within 120 feet. All creatures in the area take @Damage[(floor((@actor.level -6)/3)+2)d6[bludgeoning]|options:area-damage] damage with a @Check[fortitude|against:kineticist|basic|options:area-effect] save against your class DC. Lift any creature that fails its save to any height you choose within the area, move it up to 5 feet in any direction, then drop it. It takes falling damage normally unless it has a fly Speed.</p><hr /><p><strong>Level (+3)</strong> The rain damage increases by 1d6, and the cylinder's height increases by 5 feet.</p>"
+            "value": "<p>A hurricane lifts your enemies into the air before bringing them crashing down in a bloody rain. Your hurricane appears in a cylinder that's [[(40+(5*floor((@actor.level -6)/3)))]] feet tall, has a @Template[cylinder|distance:15]{30-foot diameter}, and is within 120 feet. All creatures in the area take @Damage[(floor((@actor.level -6)/3)+2)d6[bludgeoning]|options:area-damage] damage with a @Check[fortitude|against:kineticist|basic|options:area-effect] save against your class DC. Lift any creature that fails its save to any height you choose within the area, move it up to 5 feet in any direction, then drop it. It takes falling damage normally unless it has a fly Speed.</p><hr /><p><strong>Level (+3)</strong> The rain damage increases by 1d6, and the cylinder's height increases by 5 feet.</p>"
```

Comment:
If it's a concern, this will go below 40 feet if the character is below level 6.